src/main/java/xyz/vegaone/easytrackingv2/service/ProjectService.java

```diff
@@ -73,10 +73,18 @@ public Project updateProject(Project project) {
         return savedProject;
     }
 
-    public List<Project> findAllProjects() {
-        List<ProjectEntity> projectEntityList = projectRepo.findAll();
+    public List<Project> findAllProjects(Boolean brief) {
 
-        return projectMapper.domainToDtoList(projectEntityList);
+        if (brief) {
```

Comment:
The `if` and `else` statements should be reversed. If brief is true than we want to fetch only a small part of each project. If brief is false then fetch the projects with all their fields.